### PR TITLE
XAMARIN-42859 Load More template support added in UG

### PR DESCRIPTION
**Pull request by jeya-kasipandi**
Load more template support added in UG

### CODE REVIEW

File: Xamarin/AutoComplete/Maximum-display-item-with-Expander.md [hunk 1]
```diff
@@ -121,6 +121,96 @@ autoComplete.LoadMore(5);
...
+            };
+
+            stackLayout.Children.Add(autoComplete);
+            this.Content = stackLayout;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LoadMoreTemplate](images/Maximum-display-item-with-Expander/LoadMoreTemplate.png)
```
Comment (Choza-rajan) [inline Xamarin/AutoComplete/Maximum-display-item-with-Expander.md:212]: In the image the text is not in vertically center

Comment (PaulAnderson02) [reply Xamarin/AutoComplete/Maximum-display-item-with-Expander.md:212]: Image is very big

File: Xamarin/ComboBox/Maximum-display-item-with-Expander.md [hunk 2]
```diff
@@ -122,6 +122,97 @@ comboBox.LoadMore(5);
...
+            };
+
+            stackLayout.Children.Add(comboBox);
+            this.Content = stackLayout;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LoadMoreTemplate](images/Maximum-display-item-with-Expander/LoadMoreTemplate.png)
```
Comment (PaulAnderson02) [inline Xamarin/ComboBox/Maximum-display-item-with-Expander.md:214]: Image is very big

File: Xamarin/AutoComplete/Maximum-display-item-with-Expander.md [hunk 3]
```diff
@@ -121,6 +121,96 @@ autoComplete.LoadMore(5);
 {% endhighlight %}
 {% endtabs %}
 
+## Load more view customization
+
+SfAutoComplete allows customizing User Interface(UI) of Load More view.
```
Comment (laksmipriyamurugesan) [inline Xamarin/AutoComplete/Maximum-display-item-with-Expander.md:126]: `SfAutoComplete` allows customizing User Interface(UI) of Load More view.

File: Xamarin/AutoComplete/Maximum-display-item-with-Expander.md [hunk 4]
```diff
@@ -121,6 +121,96 @@ autoComplete.LoadMore(5);
...
+
+To customize the load more text, add the custom UI in the `LoadMoreTemplate` API in SfAutoComplete as shown in below code snippet.
```
Comment (laksmipriyamurugesan) [inline Xamarin/AutoComplete/Maximum-display-item-with-Expander.md:128]: To customize the load more text, add the custom UI in the `LoadMoreTemplate` API in `SfAutoComplete`, as shown in the following code sample.

File: Xamarin/ComboBox/Maximum-display-item-with-Expander.md [hunk 5]
```diff
@@ -122,6 +122,97 @@ comboBox.LoadMore(5);
 {% endhighlight %}
 {% endtabs %}
 
+## Load more view customization
+
+SfComboBox allows customizing User Interface(UI) of Load More view.
```
Comment (laksmipriyamurugesan) [inline Xamarin/ComboBox/Maximum-display-item-with-Expander.md:127]: `SfComboBox` allows customizing User Interface(UI) of Load More view.

File: Xamarin/ComboBox/Maximum-display-item-with-Expander.md [hunk 6]
```diff
@@ -122,6 +122,97 @@ comboBox.LoadMore(5);
...
+
+To customize the load more text, add the custom UI in the `LoadMoreTemplate` API in SfComboBox as shown in below code snippet.
```
Comment (laksmipriyamurugesan) [inline Xamarin/ComboBox/Maximum-display-item-with-Expander.md:129]: To customize the load more text, add the custom UI in the `LoadMoreTemplate` API in `SfComboBox,` as shown in the following code sample.

**Review by Choza-rajan — APPROVED** [review]
Fine

**Pull request merged**